tools/accuracy_checker/accuracy_checker/evaluators/quantization_model_evaluator.py

```diff
@@ -141,9 +141,10 @@ def _process_ready_predictions(batch_raw_predictions, batch_identifiers, batch_m
                         batch_annotation, batch_predictions, batch_meta, dump_prediction_to_annotation
                     )
                     if dump_prediction_to_annotation:
-                        self._dumped_annotations.extend([prediction.to_annotation() for prediction in predictions])
+                        annotations = [prediction.to_annotation() for prediction in predictions]
+                        self._dumped_annotations.extend(annotations)
                     metrics_result = None
-                    if not dump_prediction_to_annotation and self.metric_executor:
```

Comment:
do you really need calculate metrics if prediction == annotation?

Reply:
Yes. +we plan to implement more complex logic of annotations creation, so predictions won't always fully match annotations